radio/src/audio_arm.h

```diff
@@ -116,6 +118,7 @@ enum AudioBufferState
 struct AudioBuffer {
   audio_data_t data[AUDIO_BUFFER_SIZE];
   uint16_t size;
+  uint8_t  state;
```

Comment:
indeed, I will also change the #ifdef and introduce AUDIO_DOUBLE_BUFFERING rather than a board type